srv/methods.go

```diff
@@ -54,6 +58,121 @@ func (s *APIServer) jrpcMethods() jrpc.MethodMap {
 
 }
 
+type ResultGlobalRichList struct {
+	Address string `json:"address"`
+	Equiv   uint64 `json:"pusd"`
+}
+
+func (s *APIServer) getGlobalRichList(data json.RawMessage) interface{} {
+	params := ParamsGetGlobalRichList{}
+	_, _, err := validate(data, &params)
+	if err != nil {
+		return err
+	}
+
+	if params.Count == 0 {
+		params.Count = 100
+	}
+
+	height := s.Node.GetCurrentSync()
+	rates, err := s.Node.Pegnet.SelectRates(nil, height)
```

Comment:
This won't work if the last block does not have rates. This is less likely to happen on the mainnet, since miners only skipped one block since the start. But on local nets, if you turn off mining, there is no rates at the height.

Our `SelectMostRecentRatesBeforeHeight` uses a sql tx, I would make a similar query that does not and pass in `height+1`. Maybe return the height the rates were used from in the response too. 
Be sure not to use sql tx though, that has some write/read concurrency issues we discovered and need to resolve later.

Reply:
i changed selectmostrecentratesbeforeheight to use queryable and i'm just using that vs making a new function